config/locales/sv.yml

```diff
@@ -1,8 +1,8 @@
 ---
 sv:
   about:
-    about_hashtag_html: Dessa är offentliga toots märkta med <strong>#%{hashtag}</strong>. Du kan interagera med dem om du har ett konto någonstans i federationen.
     about_mastodon_html: Mastodon är ett socialt nätverk baserat på öppna webbprotokoll och gratis, öppen källkodsprogramvara. Det är decentraliserat som e-post
+    about_hashtag_html: Dessa är offentliga toots märkta med <strong>#%{hashtag}</strong>. Du kan interagera med dem om du har ett konto någonstans i federationen.
```

Comment:
Why did you move this message?

Reply:
Freaky fingers allowed me to add the line again, maybe wrong line then.

Reply:
The messages of the locale file must be arranged in alphabetical order. Can you return to the original position?